docs/source/API/core/parallel-dispatch/parallel_reduce.rst

```diff
@@ -93,8 +93,8 @@ Requirements:
 
   - If ``ExecPolicy::work_tag`` is ``void``, the overload without a ``WorkTag`` argument is used.
   - ``N`` must match ``ExecPolicy::rank``.
-* If the ``functor`` is a lambda, ``ReducerArgument`` must satisfy the ``Reducer`` concept or ``ReducerArgumentNonConst`` must be a POD type with ``operator +=`` and ``operator =`` or a ``Kokkos::View``.  In the latter case, the default ``Sum`` reduction is applied. If provided, the ``init``/ ``join``/ ``final`` member functions must not take a ``WorkTag`` argument even for tagged reductions.
-* If ``ExecPolicy`` is ``TeamThreadRange`` a "reducing" ``functor`` is not allowed and the ``ReducerArgument`` must satisfy the ``Reducer`` concept or ``ReducerArgumentNonConst`` must be a POD type with ``operator +=`` and ``operator =`` or a ``Kokkos::View``.  In the latter case, the default ``Sum`` reduction is applied.
+* If the ``functor`` is a lambda, ``ReducerArgument`` must satisfy the ``Reducer`` concept or ``ReducerArgumentNonConst`` must be a POD type with ``operator +=`` and ``operator =`` or a ``Kokkos::View``.  In the latter case, a sum reduction is applied where the identity is assumed to be given by the default constructor of the value type (and not by ``reduction_identity```). If provided, the ``init``/ ``join``/ ``final`` member functions must not take a ``WorkTag`` argument even for tagged reductions.
+* If ``ExecPolicy`` is ``TeamThreadRange`` a "reducing" ``functor`` is not allowed and the ``ReducerArgument`` must satisfy the ``Reducer`` concept or ``ReducerArgumentNonConst`` must be a POD type with ``operator +=`` and ``operator =`` or a ``Kokkos::View``.  In the latter case, a sum reduction is applied where the identity is assumed to be given by the default constructor of the value type (and not by ``reduction_identity```).
```

Comment:
I think it can be clearer to mention the case where `reduction_identity` is used and when `default constructor value` is.
For a user perspective it is not even clear why `identity` is needed. Are you talking about initial value for the reduction or for some other operations in the algorithms?

Reply:
The initial value is the identity (internally) so that we don't have to treat the first iteration as special.

Reply:
**Path forward:**
- accept as is, and document future behavior once refactor is in place